product_image_tab/README.md

```diff
@@ -0,0 +1,2 @@
+This module adds a new tab to the product screen where the image is displayed
+in full size.
```

Comment:
could you update the README to use the template provided here: https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst